terraform/modules/redis/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0"
+    }
+  }
+}
+
+/// elasticache cluster
```

Comment:
This comment doesn't really add anything - please consider removing.

Reply:
ok